scripts/deploy_gce_ci.sh

```diff
@@ -4,37 +4,39 @@
 #set -o nounset
 #set -o xtrace
 
-export PROJECT_NAME_CI=trillian-opensource-ci
-export CLUSTER_NAME_CI=trillian-opensource-ci
-export CLOUDSDK_COMPUTE_ZONE=us-central1-a
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+export PROJECT_NAME=trillian-opensource-ci
+export CLUSTER_NAME=trillian-opensource-ci
+export REGION=us-central1
+export ZONE=us-central1-a
+export CONFIGMAP=trillian-opensource-ci.yaml
```

Comment:
Move this chunk out into a `config.sh`-like file?

Reply:
This script is specifically for Travis to deploy to the GCE CI, I guess we could strip it out (and  possibly just set this stuff in Travis env vars), but anyone else relying on it with their forked repo would presumably edit and commit.